Common/src/interpolation_structure.cpp

```diff
@@ -1028,61 +1106,89 @@ void CMirror::Set_TransferCoeff(CConfig **config) {
 

```

Comment:
Yep, if Isoparametric is working I believe CMirror should also do. I don't have any right now, but I may be using it soon. In that case I'll let you know if I find any issue or if it works as expected. It was just to know if it was tested, thanks!